basil/TL/Serial.py

```diff
@@ -71,4 +71,4 @@ def _readline(self):  # http://stackoverflow.com/questions/16470903/pyserial-2-6
             data += character
             if not character:
                 break
-        return data
+        return data.decode('utf-8')
```

Comment:
I am not sure this can be done. There might be bytes returned that have to representation in utf-8, no? Did you check this?

Reply:
i only checked with a TTi, a Keithley sand the probestation via serial. Those work fine...

Reply:
Seems like utf-8 is a way to go https://pyserial.readthedocs.io/en/latest/pyserial_api.html#serial.Serial.write

@DavidLP We should merge this and see.